Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGreyExampleSwiftTests.swift

```diff
@@ -24,33 +24,33 @@ class EarlGreyExampleSwiftTests: XCTestCase {
   func testBasicSelection() {
     // Select the button with Accessibility ID "clickMe".
     // This should throw a warning for "Result of Call Unused."
-    EarlGrey.select(elementWithMatcher: grey_accessibilityID("ClickMe"))
+    EarlGrey.selectElement(with: grey_accessibilityID("ClickMe"))
```

Comment:
Could all the swift changes go in another PR where you update the syntax?

Reply:
It kind of tedious to update the syntax without modifying other changes unless I update the script temporarily and rerun it. But then I will have to redo all other manual changes. I'd rather put them in the same PR for now and make some notes in the PR.